src/diagnostic-channel-publishers/tests/mongodb.spec.ts

```diff
@@ -57,11 +38,21 @@ describe("mongodb", function() {
 
         const z1 = Zone.current.fork({name: "1"});
         z1.run(() =>
-        mongodb.MongoClient.connect("mongodb://localhost:27017/testdb", function(err, db) {
+        mongodb.MongoClient.connect("mongodb://localhost:27017", { useNewUrlParser: true }, function(err, client) {

```

Comment:
Why is useNewUrlParser necessary? We should also validate nothing is broken the way the test worked before

Reply:
It just generates warnings when running tests, tests passed either way. The parsing results should (hopefully) not be affected at all by this setting.